Sources/StrongContractResponse/All/StrongContract.handle.swift

```diff
@@ -10,19 +10,18 @@ import Vapor
 
 public extension StrongContractClient.Request {
 
-    typealias PayloadToResponse = (Payload, Vapor.Request) async throws -> ResponseAdaptor
-
+    typealias Handler = (Payload, Vapor.Request) async throws -> ResponseAdaptor
```

Comment:
Maybe PayloadToAdaptor? and the new one can be PayloadToResponse now

Reply:
PayloadToResponseAdaptor is fine.

Reply:
The reason I named it that way is so that it is clear "at a glance" what it is doing.  Taking a payload, and returning a response.  The adaptor part is really a carrier for the response to retain the type information.  Thats why I didn't think it was important for the name.  It makes it clear that this is for the server side when the client sends a piece of information and expects a response.